packages/orbit-components/src/tmp-tailwind/Select/index.tsx

```diff
@@ -0,0 +1,230 @@
+              ? "[&>.orbit-input-field-fake-input]:hover:shadow-form-element-error-hover"
+              : "[&>.orbit-input-field-fake-input]:hover:shadow-form-element-hover"),
+          "focus-within:outline-none",
+          "[&_.orbit-input-field-fake-input]:focus-within:outline-blue-normal [&_.orbit-input-field-fake-input]:focus-within:outline [&_.orbit-input-field-fake-input]:focus-within:outline-2",
+        )}
+      >
+        {label && inlineLabel && (
+          <div
+            className={cx(
+              "pointer-events-none z-[3] flex h-full items-center justify-center",
+              error || help ? "ps-xxs" : "ps-sm",
+              "[&_.orbit-form-label]:text-normal [&_.orbit-form-label]:mb-0 [&_.orbit-form-label]:inline-block [&_.orbit-form-label]:max-w-[20ch] [&_.orbit-form-label]:overflow-hidden [&_.orbit-form-label]:text-ellipsis [&_.orbit-form-label]:whitespace-nowrap [&_.orbit-form-label]:leading-normal",
+            )}
+            ref={labelRef}
+          >
+            {help && !error && (
+              <span className="flex" ref={iconRef}>
+                <InformationCircle color="info" size="small" />
+              </span>
+            )}
+            {error && (
+              <span className="flex" ref={iconRef}>
+                <AlertCircle color="critical" size="small" />
+              </span>
+            )}
+            <FormLabel
+              filled={!!value}
+              required={required}
+              error={!!error}
+              help={!!help}
+              inlineLabel={inlineLabel}
+            >
+              {label}
+            </FormLabel>
+          </div>
+        )}
+
+        <div className="relative z-[3] h-full w-full">
+          {prefix && (
+            <div className="px-sm pointer-events-none absolute top-0 z-[3] flex h-full items-center">
+              {prefix}
+            </div>
+          )}
+          {customValueText && (
+            <div
+              className={cx(
+                (disabled && "text-ink-light") || filled ? "text-ink-dark" : "text-ink-light",
+                "font-form-element-normal font-base pointer-events-none absolute bottom-0 top-0 z-[3] flex items-center",
+                prefix ? "ps-[48px]" : "ps-sm",
+              )}
+            >
+              {customValueText}
+            </div>
+          )}
+          <select
+            className={cx(
+              "cursor-pointer appearance-none bg-transparent outline-none",
+              filled ? "text-form-element-label-foreground" : "text-form-element-foreground",
+              "font-base text-form-element-normal",
+              "pe-xxl",
+              prefix ? "ps-[48px]" : "ps-sm",
+              "shrink grow basis-1/5",
+              "h-full w-full",
+              "border-0",
+              customValueText &&
+                "!text-transparent [&:-internal-autofill-selected]:[-webkit-text-fill-color:transparent] [&:-webkit-autofill]:[-webkit-text-fill-color:transparent]",
```

Comment:
It's hard to know under which circumstances these classes are applied.

```css
${customValueText &&
  `
    &:-webkit-autofill,
    &:-internal-autofill-selected {
      -webkit-text-fill-color: transparent;
    `}
`}
```

The only thing I can assure is that they are present in the CSS bundle:


```css
.\[\&\:-internal-autofill-selected\]\:\[-webkit-text-fill-color\:transparent\]:-internal-autofill-selected {
  -webkit-text-fill-color: transparent
}

.\[\&\:-webkit-autofill\]\:\[-webkit-text-fill-color\:transparent\]:-webkit-autofill {
  -webkit-text-fill-color: transparent
}
```

The [autofill pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/:autofill) are supposed to be used only in `input` elements if I'm not wrong. So I doubt these can have any effect on a `select` element.

Do you perhaps know how to test it better, or if they are actually needed?

Reply:
I think it's a very specific selector. In order to test that, you will need to properly create a completely functional select in the storybook and try the browser autofill, even then I am not sure if it will be visible. 

I think we can remove it completely, wdyt?

Reply:
I also think we can remove it. In any case, we're setting the color to text-transparent when there's a `customValueText`.